wa/workloads/rt_app/__init__.py

```diff
@@ -231,7 +231,13 @@ def _load_json_config(self, context):
         config_file = self._generate_workgen_config(user_config_file,
                                                     context.output_directory)
         with open(config_file) as fh:
-            config_data = json.load(fh, object_pairs_hook=OrderedDict)
+            try:
+                config_data = json.load(fh, object_pairs_hook=OrderedDict)
+            except ValueError:
+                # We were not able to parse the JSON file. Raise an informative error.
+                msg = "Failed to parse {}. Please make sure it is valid JSON."
+                raise ConfigError(msg.format(user_config_file))
```

Comment:
Maybe format the `ValueError` into the message as well (if it gives more information about the syntax error)?

Reply:
The ValueError message was:

`ValueError(No JSON object could be decoded)`

Which does not seem to be overly informative so I personally feel that it is not useful.

Reply:
Agreed. I'd hoped it'd at least give a line number or something. This is fine in that case.